redbadger/website-honestly
# #157: Fix Missing Page Titles

### Motivation

- Closes #157 for missing page titles.  Had a spare 20 mins to take a quick look to fix the issue I raised.

### Test plan

* Go to home page, check page title `Home | Red Badger`
* Go to what we do, check page title `What we do | Red Badger`
* Go 404 page, check page title `Not found | Red Badger`

### Pre-merge checklist


@lpil (Looking for guidance on this one, compiler `layoutTemplate` only returns body content, not sure I understand it fully... yet).
